docs/api.rst

```diff
@@ -10828,6 +10830,66 @@ Querying  NFTs
    :statuscode 502: An external service used in the query such as opensea could not be reached or returned unexpected response.
 
 
+Querying a single NFT
+=====================
+
+.. http:post:: /api/(version)/nfts
```

Comment:
For this, I think we should follow the guide from this issue. https://github.com/rotki/rotki/issues/5009
Which is to make `/assets/mappings` and `/assets/search/levenshtein` endpoints could also handle NFT information, instead of creating new endpoint

Reply:
While originally the spec was not in the issue, I added it two weeks ago, just before I left for vacation because @nebolax wrote on discord that it was not clear what needed to be done. So I went and added the spec and pinged when the issue was updated. 

I can't understand how stuff can be missing, it's like nobody checked the issue properly.